src/pint/models/pulsar_binary.py

```diff
@@ -211,11 +211,14 @@ def update_binary_object(self, toas=None, acc_delay=None):
                 aliase = []
 
             if hasattr(self, par) or list(set(aliase).intersection(self.params)) != []:
-                try:
-                    pint_bin_name = self.match_param_aliases(par)
-                except ValueError:
+                pint_bin_name = self.match_param_aliases(par)
```

Comment:
Shouldn't `match_param_aliases` raise an exception if nothing is found? That would simplify this code and avoid having to check every time the function is called.

Reply:
This part will have to be addressed in #1042 first. It will have a conflict here, once it gets merged.